src/api/controllers/plugin.rs

```diff
@@ -5,10 +5,11 @@
 use crate::{
```

Comment:
Do we want to have maximum timeout for plugin to complete? like 5min? Maybe we should start with default and allow people to override.